examples/lessons/13/app.js

```diff
@@ -233,6 +244,7 @@ const animationLoop = new AnimationLoop({
 
     moon
       .setUniforms({
+        uSampler: moonTexture,
```

Comment:
Do we need to set this again if we are no longer sharing programs?

Teaching best practices would be to only set uniforms that vary each frame?

Reply:
No, that was to work around the program-sharing issue.